### ISSUE TITLE
~ 0.1% of smb_mapping events have no timestamp

### ISSUE THREAD

**Issue by vbr-cts** [zeek/zeek#1788]
Hi.

Using zeek 3.2.2.
Out of a ~100K sample of my smb_mapping log , around 100 do not have a timestamp. The rest look perfectly fine.

Examples of said events:

```
{"uid":"COvrT54v8bNFK2aojb","id.orig_h":"192.168.xxx.xxx","id.orig_p":3470,"id.resp_h":"192.168.xxx.xxx","id.resp_p":445,"service":"IPC","share_type":"PIPE"}
{"uid":"CYwhPB4B8zYv85A8Kk","id.orig_h":"192.168.xxx.xxx","id.orig_p":4646,"id.resp_h":"192.168.xxx.xxx","id.resp_p":445,"service":"A:","native_file_system":"NTFS","share_type":"DISK"}
```

Does anyone know what might be causing this ?

**Comment by rsmmr** [comment]
Any chance you can provide a pcap for some of those instances?

**Comment by JustinAzoff** [comment]
This was fixed in 4.1.0

```
commit db7aba0835ef82e16cf56017a876f61ab8b4a72a
Author: Seth Hall <seth@corelight.com>
Date:   Wed Mar 10 13:52:44 2021 +0000

    ts fields in SMB logs now default to network_time()
    
    This avoids a problem identified by amanbansal2709 in pull
    request #1288. I fixed it in a different way than that pull request
    by making sure the ts field is always set so that this isssue doesn't
    return in the future.
```

Can probably backport that commit to 3.2.2, or just upgrade :-)

**Comment by vbr-cts** [comment]
Awesome. Upgrade is planned pretty soon so no need to backport. Thanks a lot.
